src/renderer/containers/UploadSelectionPage/DragAndDropPrompt/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { UploadOutlined } from '@ant-design/icons';
+import { Button } from 'antd';
+import { OpenDialogOptions, ipcRenderer } from "electron";
+import { isEmpty } from "lodash";
+import React from 'react';
+
+import { RendererProcessEvents } from '../../../../shared/constants';
+import { UploadType } from '../../../types';
+
+const styles = require("./styles.pcss");
+
+interface DragAndDropPromptProps {
+    openDialogOptions: OpenDialogOptions;
+    onDrop: (files: string[]) => void;
+    uploadType: UploadType
+}
+
+export default function DragAndDropPrompt(props: DragAndDropPromptProps) {
+
+    const onBrowse = async () => {
+        const filePaths = await ipcRenderer.invoke(
+          RendererProcessEvents.SHOW_DIALOG,
+          props.openDialogOptions
+        );
+    
+        // If cancel is clicked, this callback gets called and filePaths is undefined
+        if (filePaths && !isEmpty(filePaths)) {
+          props.onDrop(filePaths);
+        }
+    };
+
+    let helpText: string | null = null;
+    if (props.uploadType) {
+      helpText = props.uploadType === UploadType.File
+        ? "Accepted file formats include: .czi, .tiff, .ome.tiff, .csv, ..."
+        : "Accepted file formats include: .zarr, .sldy, ..."
```

Comment:
Is the casing correct on `.sldy` ?  I see instances of both all lower and all upper in the wild.

Reply:
They look like they're lowercase `.sldy` under Jackie's folder on the Vast right now, but yeah I've been a bit confused by that.